### DISCUSSION TITLE
Topological view based on selection

### DISCUSSION THREAD

**Discussion by torbensp** [mattieserver/netbox-topology-views#367] [Q&A]
First of all I would like to say that this plug-in is really nice and very useful as it gives a very good overview of a network with a lot of selection / filter options. The system that I am implementing in Netbox contains many hundreds Cisco IOS devices together with end devices such as IP telephones, WIFI AP's, CCTV cameras etc. The whole fiber and copper infrastructure, including FO and LAN patch panels are also documented in Netbox.

My question is related to how I can define filters whereby I only get particular connection and associated devices shown in the topological view (e.g. the CCTV system should only show CCTV cameras, relevant cables, patch panels and switch / switch ports that connects to the cameras).

For the end devices, it is quite simple as I can use "tag" or perhaps "tenant" but if I have a patch panel or a switch, which is common to several systems like CCTV, telephones and WIFI AP's, how can I do so only cables, connections and relevant devices are shown? At the moment many connections are shown between a switch and a patch panel even if I try to define a specific tag on a particular port on the switch and patch panel.

Any ideas and suggestion is much appreciated!

Best regards,

Torben

**Comment by dreng** [comment]
Hi Torben,

Not sure if I understood you correctly. Basically all connections of the filtered units are displayed. It is currently not possible to display only some connections (aka cables) for a device.
